src/Microsoft.AspNet.Mvc.Core/FilterActionInvoker.cs

```diff
@@ -219,7 +368,36 @@ private IFilter[] GetFilters()
         private async Task InvokeAllActionFiltersAsync()
         {
             _cursor.SetStage(FilterStage.ActionFilters);
+
+            Debug.Assert(_resourceExecutingContext != null);
+            var bindingContext = new ActionBindingContext()
```

Comment:
Things which we would not want a ResourceFilter to touch but are needed in binding would make sense only in ABC and not in ResourceBindingContext (For ex InputFormatterSelector).